core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -78,8 +78,11 @@ public ImportExtendedMutationData(File mutationFile, int geneticProfileId, Strin
         this.geneticProfileId = geneticProfileId;
         this.swissprotIsAccession = false;
         this.genePanel = genePanel;
-        this.filteredMutations = filteredMutations;
-
+        for (String filteredMutation : (Arrays.asList(filteredMutations.split(",")))) {
+            filteredMutation = filteredMutation.trim();
+            this.filteredMutations.add(filteredMutation);
+        }
```

Comment:
Actually, I also think this would be better done in `getVariantClassificationFilter` immediately. This makes it clear that this setting is about a set of values.